pulsar-broker-common/src/main/java/org/apache/pulsar/bookie/rackawareness/BookieRackAffinityMapping.java

```diff
@@ -161,7 +161,7 @@ private String getRack(String bookieAddress) {
         }
 
         if (bi != null
-                && !Strings.isNullOrEmpty(bi.getRack())
+                && !com.google.api.client.util.Strings.isNullOrEmpty(bi.getRack())
```

Comment:
I think we can use  "org.apache.commons.lang3.StringUtils#isEmpty" to avoid this checkstyle issue.